### PR TITLE
Ma 4014/create lib files

### PR DESCRIPTION
**Pull request by mayankbharadwaj**
Added two new files as part of our aws-service-lib promise library.

### CODE REVIEW

File: aws_promise/awsCfnStack.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+'use strict'
+var AWS = require('aws-sdk');
```
Comment (SonalOjha) [inline aws_promise/awsCfnStack.js]: change this to const

File: aws_promise/iam.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+var AWS = require('aws-sdk');
```
Comment (SonalOjha) [inline aws_promise/iam.js]: change this to const

**Review by SonalOjha — CHANGES_REQUESTED** [review]

File: aws_promise/awsCfnStack.js (hunk 1 above)
Comment (mayankbharadwaj) [reply aws_promise/awsCfnStack.js]: done

File: aws_promise/iam.js (hunk 2 above)
Comment (mayankbharadwaj) [reply aws_promise/iam.js]: done

File: aws_promise/awsCfnStack.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+  findStackOutputs: function(input) {
+    const cfn = this.findService(input);
+    let params = {
+        StackName: input.stackName
+    };
+    return cfn.describeStacks(params).promise();
+}
+
+}
```
Comment (bhargavapratik) [inline aws_promise/awsCfnStack.js]: What is it? can you remove this.

Comment (SonalOjha) [reply aws_promise/awsCfnStack.js]: This should not be removed, not indented properly.

**Review by SonalOjha — APPROVED** [review]
LGTM

**Review by RajkumarShivage — APPROVED** [review]
LGTM

**Pull request merged**